README.md

```diff
@@ -20,7 +20,7 @@ import { oas } from 'koa-oas3';
 
 const app = new Koa();
 app.use(bodyParser());
-app.use(oas({
+app.use(await oas({
```

Comment:
I'd extract this to a variable as an example, might be better for the compiler

Reply:
Nah, every request shares the same middleware that's created from this `await oas()` function.